simapp/testutil_network_test.go

```diff
@@ -1,27 +1,63 @@
 package simapp_test
 
 import (
+	"math/rand"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/suite"
 
+	sdkmath "cosmossdk.io/math"
 	"cosmossdk.io/simapp"
 
+	cmtcfg "github.com/cometbft/cometbft/config"
+	servertypes "github.com/cosmos/cosmos-sdk/server/types"
 	"github.com/cosmos/cosmos-sdk/testutil/network"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	authsim "github.com/cosmos/cosmos-sdk/x/auth/simulation"
+	banksim "github.com/cosmos/cosmos-sdk/x/bank/simulation"
+	xsim "github.com/cosmos/cosmos-sdk/x/simulation"
 )
 
 type IntegrationTestSuite struct {
 	suite.Suite
 
+	cfg      network.Config
+	v0app    *simapp.SimApp
+	rand     *rand.Rand
+	accounts []simtypes.Account
+
 	network *network.Network
 }
 
 func (s *IntegrationTestSuite) SetupSuite() {
 	s.T().Log("setting up integration test suite")
 
+	s.rand = rand.New(rand.NewSource(42))
+	params := xsim.RandomParams(s.rand)
+	s.accounts = simtypes.RandomAccounts(s.rand, params.NumKeys())
+	// TODO: improve this hack to get at the bank keeper.
+	s.cfg = network.DefaultConfig(func() network.TestFixture {
+		fixture := simapp.NewTestNetworkFixture()
+		appCtr := fixture.AppConstructor
+		fixture.AppConstructor = func(val network.ValidatorI) servertypes.Application {
+			app := appCtr(val)
+			s.v0app = app.(*simapp.SimApp)
+			return app
+		}
+		return fixture
+	})
+	s.cfg.NumValidators = s.rand.Intn(10)
```

Comment:
```go
// TODO(@elias-orijtech): allow number of validators to
// be programmatically passed in perhaps via cli arguments
```

Reply:
There has to be some limit? I assume this particular parameter will be revisited once testutil runs on top of MockComet.